aries_vcx/src/common/primitives/mod.rs

```diff
@@ -117,7 +117,7 @@ pub mod integration_tests {
     #[tokio::test]
     #[ignore]
     async fn test_pool_from_pool_ledger_with_id() {
```

Comment:
I know you didn't touch this, but let's give this a better name, perhaps `test_pool_create_and_get_schema`

Reply:
Done